### DISCUSSION TITLE
Blind Level Action fehlt in Gruppenaktion

### DISCUSSION THREAD

**Discussion by tofrie** [Zefau/ioBroker.jarvis#1507] [Q&A] [answered]
In der Jarvis v2 gab es die Gruppenfunktionen hoch und runter, diese wird bei mir in v3 nicht angezeigt…

![43A2D22A-C944-4094-8C83-987BF468736B](https://user-images.githubusercontent.com/60553702/149661042-929c1f20-07ec-4146-b19e-81c54760e53e.png)

**Comment by Zefau** [comment]
Es ist `activity` (Boolean) in den Component Options auszuwählen, siehe https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/beispiele/zusatzprogramme/blind#blindlevelaction-zum-level-dp

**Comment by tofrie** [comment]
Klappt leider nicht. Unter Group Element kann ich Blind Level Action nicht auswählen.

**Reply by Zefau** [reply]
Achso sorry, hab mich verlesen. BlindLevelAction gibt nicht als Group Element in v3. Das muss ich in v3.1 nachliefern, sorry.

**Comment by SKB-CGN** [comment]
Hat sich hier zufällig schon etwas getan?

**Reply by mcuiobroker** [reply]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/geraet/actionelement/blindlevelaction#mehrere-rolllaeden-blinds-gleichzeitig-steuern
??

**Comment by Zefau** [comment, marked as answer]
mit `v3.1.4-beta.5` wieder verfügbar, siehe https://github.com/Zefau/ioBroker.jarvis/issues/2229

**Comment by Draygv2** [comment]
Man kann es jetzt zwar hinzufügen, aber da meine Markisen die umgekehrte Logik haben, ist bei mir jetzt hoch runter und runter hoch. Kann ich das noch irgendwie anpassen?

**Reply by mcuiobroker** [reply]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/styles/blindlevelaction-rolllaeden/button-tauschen

**Reply by Draygv2** [reply]
Danke für den Link. Leider bin ich zu doof dafür.
![image](https://user-images.githubusercontent.com/62559741/224568471-79c72d07-3228-478e-9469-2f1bc991ae60.png)
![image](https://user-images.githubusercontent.com/62559741/224568480-24cbda39-bb42-4034-8c50-2fbb02da15cf.png)
![image](https://user-images.githubusercontent.com/62559741/224568509-a2c9957a-c9ad-425e-900c-cd96baece414.png)
Was mache ich falsch?

**Reply by mcuiobroker** [reply]
Funktioniert anscheinend so nicht mehr. Muss ich noch schauen.

**Reply by mcuiobroker** [reply]
Man kann es momentan nur auf das Widget eingrenzen, keine Ahnung warum?
`.jarvis-StateListItem:nth-child(1)` Den Teil raus nehmen.

**Reply by Draygv2** [reply]
Jop geht und reicht mir so auch thanks.

**Comment by mcuiobroker** [comment]
@Zefau  `:nth-child` zieht nicht mehr?
